public/index.html

```diff
@@ -4,6 +4,7 @@
       <title>Learner Profile | edX</title>
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
```

Comment:
Should we do this for the other apps as well? Do you need to remove a reference to this font from somewhere else?

Reply:
It's in edx-bootstrap. I've make a PR there which has an alternative approach: https://github.com/edx/edx-bootstrap/pull/54